package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.13.0",
   "description": "Supernova action button",
   "license": "MIT",
-  "author": "QlikTech International AB",
+  "author": "Goldberg Productions",
```

Comment:
This seems like an unnecessary change :)

Reply:
updated